test/integration/api/projects.js

```diff
@@ -349,7 +355,7 @@ describe('api: /projects', () => {
           .then(({ body }) => {
             body.verbs.should.be.an.Array();
             body.verbs.length.should.be.greaterThan(39);
-            body.verbs.should.containDeep([ 'user.password.invalidate', 'project.delete' ]);
+            body.verbs.should.containDeep([ 'user.password.invalidate' ]);
```

Comment:
Why remove `project.delete`? I'm thinking that it's a stand-in for "powerful project permissions."